{{cookiecutter.folder_name}}/.metadata.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I'm not sure what the best name is - I partly put this in because @jonathansick 's publishing system needs something like it, but 'm also not sure what he's going to need. Certainly we shoudl tell people they can edit it to change the behavior of the Makefile.